bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectInverter.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="localConnectInverter">
+
+		<label>Local Connect Inverter</label>
+		<description>The inverter representation that supports local connections via HTTP</description>
+
+		<channels>
+			<channel id="inverter-output-power" typeId="system.electric-power">
+				<label>"Inverter Input/Output Power"</label>
+				<description>"Power to/from the inverter"</description>
+			</channel>
+			<channel id="inverter-current" typeId="system.electric-current">
+				<label>"Inverter Input/Output Current"</label>
+				<description>"Current to/from the inverter"</description>
+			</channel>
+			<channel id="inverter-voltage" typeId="system.electric-voltage">
+				<label>"Inverter Voltage"</label>
+				<description>"Voltage of the inverter"</description>
+			</channel>
+			<channel id="inverter-frequency" typeId="frequency">
+				<label>"Inverter Voltage Frequency"</label>
+				<description>"Voltage frequency of the inverter.(should be the same as the feed in voltage frequency)"</description>
+			</channel>
+
+			<channel id="pv1-voltage" typeId="system.electric-voltage">
+				<label>"PV String 1 Voltage"</label>
```

Comment:
I'm not sure what "String" means here, but perhaps for the label it could be left out to keep it short?
```suggestion
				<label>"PV 1 Voltage"</label>
```